### DISCUSSION TITLE
Game crashed after randomly holding "C" after gameplay

### DISCUSSION THREAD

**Discussion by rvnexe** [ppy/osu#23865] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

Hi, I was in gameplay of [This ](https://lazer.ppy.sh/beatmapsets/1828516#osu/3788435)beatmap (Habbes's Hyper)
And I reached at the end of the beatmap. I held "C" key (Smoke) and drew some lines,
Then game freezed and crashed to desktop.

Just for Information, My play details were submitted in the lazer.ppy.sh (Crash was after submission)

### Version

2023.605.0-lazer

### Logs

[network.log](https://github.com/ppy/osu/files/11709717/network.log)
[performance.log](https://github.com/ppy/osu/files/11709718/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11709719/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11709720/updater.log)
[database.log](https://github.com/ppy/osu/files/11709721/database.log)
[input.log](https://github.com/ppy/osu/files/11709722/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11709723/legacy-ipc.log)

**Comment by rvnexe** [comment]
![image](https://github.com/ppy/osu/assets/89677246/c29f0576-2e4b-489b-b5c1-22765afcabdb)
This is what i drew probably

**Comment by bdach** [comment]
Logs attached look clean, no error in sight (although they were cut off abruptly). Please check windows event viewer for any related errors.

**Comment by rvnexe** [comment]
Here:  (Rename it: remove .txt from its end)
[osu-crash.evtx.txt](https://github.com/ppy/osu/files/11711975/osu-crash.evtx.txt)

Also, I got backup from client.realm etc. . Can they be helpful?

**Comment by bdach** [comment, marked as answer]
That looks like a generic "access violation" error, suggesting a driver issue.

Can you try upgrading your graphics card drivers to the newest version, if they aren't already?

**Reply by rvnexe** [reply]
Maybe. thank you anyway
